em-app/src/platform/mod.rs

```diff
@@ -0,0 +1,11 @@
+#[cfg(feature = "sgx")]
```

Comment:
I am not sure how thorough we should be with copyright, but in any case, copyright is missing from a few files.